### ISSUE TITLE
Is there any plans to support OS X 10.11 environment

### ISSUE THREAD

**Issue by ptmt** [travis-ci/travis-ci#5015]
I have the [following error](https://travis-ci.org/ptmt/neutrino) and can't figure out how to fix it to pass on OS X 10.10.5: 

```
Failed to query the list of test cases in the test bundle: dyld: 
  Symbol not found: ___NSArray0__
```

**Comment by joshk** [comment]
Hi @ptmt, thanks for opening the issue.

We have 10.11 support in the works and will update this ticket when we have more information.

**Comment by mattyohe** [comment]
@ptmt of note: 10.11 support is available with the 7.2 beta: http://docs.travis-ci.com/user/languages/objective-c/#Xcode-7.2-Beta

**Comment by ptmt** [comment]
@mattyohe Thanks, that solved the problem!
